src/status_im/signals/core.cljs

```diff
@@ -12,14 +12,23 @@
 (fx/defn status-node-started
   [{db :db :as cofx}]
   (fx/merge cofx
-            {:db (assoc db :status-node-started? true)}
-            #(when (:password (accounts.db/credentials cofx))
+            (merge
```

Comment:
`(let [{:node/keys [restart? address]} db
        can-login? (and (not restart? (:password (accounts.db/credentials cofx)))] ...`